spec/commands/v2/import_spec.rb

```diff
@@ -36,6 +36,11 @@
       }
     end
 
+    before do
```

Comment:
Since we're having to duplicate this in a lot of places, I wonder if there is some way we can improve this. Perhaps have it in a class method called `GoogleAnalyticsTestHelper` or similar, and then use `include GoogleAnalyticsTestHelper` in the tests which need it?